projects/github.com/mamba-org/mamba/package.yml

```diff
@@ -23,14 +23,22 @@ build:
         curl -Lfo mamba.sh "https://github.com/conda-forge/miniforge/releases/download/22.11.1-4/Mambaforge-22.11.1-4-$PLATFORM.sh" #FIXME UGLY HACK, NEED TO MOVE '-4' TO versions.anytext
     fi
 
+    sleep 1
     pwd
+    sleep 1
     ls
 
     # install mamba
     chmod +x mamba.sh
     ./mamba.sh $ARGS #FIXME when running on gh actions, this fails
                      # ^ /opt/github.com/mamba-org/mamba/v22.11.1.4/bin/python: bad interpreter: No such file or directory
```

Comment:
does it need python as a *build* dependency?

Reply:
No, it comes with it's own instance of python

This particular piece of code has been updated and no longer has a #FIXME